extensions/interactions/TextInput/text-input-prediction.service.ts

```diff
@@ -21,35 +21,50 @@
  * function on Oppia-ml.
  */
 
-require('classifiers/svm-prediction.service.ts');
-require('classifiers/text-input.tokenizer.ts');
-require('classifiers/count-vectorizer.service.ts');
-
-require('interactions/interactions-extension.constants.ajs.ts');
-
-angular.module('oppia').factory('TextInputPredictionService', [
-  'CountVectorizerService', 'SVMPredictionService',
-  'TextInputTokenizer', 'TEXT_INPUT_PREDICTION_SERVICE_THRESHOLD', function(
-      CountVectorizerService, SVMPredictionService,
-      TextInputTokenizer, TEXT_INPUT_PREDICTION_SERVICE_THRESHOLD) {
-    return {
-      predict: function(classifierData, textInput) {
-        var cvVocabulary = classifierData.cv_vocabulary;
-        var svmData = classifierData.SVM;
-
-        // Tokenize the text input.
-        textInput = textInput.toLowerCase();
-        var textInputTokens = TextInputTokenizer.generateTokens(textInput);
-
-        var textVector = CountVectorizerService.vectorize(
-          textInputTokens, cvVocabulary);
-        var predictionResult = SVMPredictionService.predict(
-          svmData, textVector);
-        if (predictionResult.predictionConfidence >
-            TEXT_INPUT_PREDICTION_SERVICE_THRESHOLD) {
-          return predictionResult.predictionLabel;
-        }
-        return -1;
-      }
-    };
-  }]);
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+import {SVMPredictionService} from 'classifiers/svm-prediction.service';
```

Comment:
Add space before and after brackets -- ditto below

Reply:
Done